### PR TITLE
Show dish info

### PR DESCRIPTION
**Pull request by Sajaas**
Waffle story: https://waffle.io/CraftAcademy/slow_food_sinatra_august_17/cards/59aff3bd21ab6a0057e87ef2

* Added feature show price
* Added feature show description

### CODE REVIEW

File: features/list_dishes_on_index.feature [hunk 1]
```diff
@@ -13,3 +13,15 @@ Scenario: user sees info about a dish if the dishes exist in database
   And I visit the site
   Then I should see a message saying "Carbonara"
   And I should see a message saying "Bolognese"
+
+Scenario: user sees the price of a dish
+  Given there is a dish named "Carbonara" in our database
+  And its dish price is "120" in our database
+  And I visit the site
+  Then I should se a message saying "120"
```
Comment (tochman) [inline features/list_dishes_on_index.feature]: We probably want to fix the spelling mistakes in steps and  step definitions

File: features/step_definitions/steps.rb [hunk 2]
```diff
@@ -9,3 +9,15 @@
 Given(/^there is a dish named "([^"]*)" in our database$/) do |dish_name|
   Dish.create(name: dish_name)
 end
+
+Given(/^its dish price is "([^"]*)" in our database$/) do |dish_price|
+  Dish.create(name: dish_price)
```
Comment (tochman) [inline features/step_definitions/steps.rb]: Well, this is not entirely correct...

Comment (tochman) [reply features/step_definitions/steps.rb]: With this the dish will have a `name`of 120 ;-)

**Review by tochman — CHANGES_REQUESTED** [review]

File: lib/views/index.erb [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+<title>Restaurant</title>
+</head>
+
+<body>
+
+<div>
+  <h1>Menu</h1>
+  <% if @dishes.any? %>
+    <% @dishes.each do |dish| %>
+      <p><h2> <%= dish.name %> </h2></p>
```
Comment (tochman) [inline lib/views/index.erb]: no need to wrap the `h2` in a `p` tag.

File: features/step_definitions/steps.rb (hunk 2 above)
Comment (Sajaas) [reply features/step_definitions/steps.rb]: Haha! Ooops, we did Dish.create(price: dish_price) before, not sure why this is the one we ended up with! x)

File: features/step_definitions/steps.rb [hunk 4]
```diff
@@ -9,3 +9,15 @@
...
+end
+
+Then(/^I should se a message saying "([^"]*)"$/) do |message|
+  expect(page).to have_content message
+end
+
+Given(/^its dish desription is "([^"]*)" in our database$/) do |dish_description|
+  Dish.create(name: dish_description)
```
Comment (tochman) [inline features/step_definitions/steps.rb]: And here the dish name would be whatever you pass in as description.

File: lib/views/index.erb (hunk 3 above)
Comment (Sajaas) [reply lib/views/index.erb]: Good point!

File: features/list_dishes_on_index.feature (hunk 1 above)
Comment (Sajaas) [reply features/list_dishes_on_index.feature]: Changed "se" to "see"!

File: features/step_definitions/steps.rb (hunk 4 above)
Comment (Sajaas) [reply features/step_definitions/steps.rb]: Changed to Dish.create(description: dish_description)!

File: features/step_definitions/steps.rb [hunk 5]
```diff
@@ -9,3 +9,15 @@
...
+Given(/^its dish price is "([^"]*)" in our database$/) do |dish_price|
+  Dish.create(price: dish_price)
+end
+
+Then(/^I should se a message saying "([^"]*)"$/) do |message|
+  expect(page).to have_content message
+end
+
+Given(/^its dish desription is "([^"]*)" in our database$/) do |dish_description|
+  Dish.create(description: dish_description)
```
Comment (tochman) [inline features/step_definitions/steps.rb]: Okay, so you fixed the right attribute problem I was pointing to. All good. But, there is another problem. ;-) 
Here you are creating a totally new dish without a name and price but with a description. What you want to do is to make sure that your "Carbonara" dis has a description right? 

so what you need to do is:
1. Find the dish with name "Carbonara" in the database
2. Update its description with whatever value you have stored in the `dish_description` variable

Not create a totally new dish. Does that make sense?

**Review by tochman — CHANGES_REQUESTED** [review]

File: features/step_definitions/steps.rb (hunk 5 above)
Comment (Sajaas) [reply features/step_definitions/steps.rb]: Something like `dish = Dish.first(:name => ‘Carbonara’)`? And `dish.update(:description => 'whatever this dish is')`?

File: features/step_definitions/steps.rb [hunk 6]
```diff
@@ -11,13 +11,15 @@
...
+  dish.update(price: dish_price)
 end
 
 Then(/^I should se a message saying "([^"]*)"$/) do |message|
   expect(page).to have_content message
 end
 
 Given(/^its dish desription is "([^"]*)" in our database$/) do |dish_description|
-  Dish.create(description: dish_description)
+  dish = Dish.first(:name => 'Carbonara')
```
Comment (tochman) [inline features/step_definitions/steps.rb]: Much better. Now I want you to update this command to the correct hash syntax ;-)
(and the one about price)
```
dish = Dish.first(name: 'Carbonara')
```

**Review by tochman — CHANGES_REQUESTED** [review]

File: features/step_definitions/steps.rb (hunk 6 above)
Comment (Sajaas) [reply features/step_definitions/steps.rb]: Ok! I used the datamapper suggestion. I'll change it! :)

**Pull request merged**